xc/xc7/tests/iosettings/CMakeLists.txt

```diff
@@ -1,50 +1,100 @@
-#add_custom_target(all_xc7_iosettings_diff_fasm)
+# These targets make Kokoro complain that the Ninja version is too low.
+#add_custom_target(all_xc7_iosettings_generate)
+#add_custom_target(all_xc7_iosettings_bin)
+#add_custom_target(all_xc7_iosettings_bit_v)
+#add_custom_target(all_xc7_iosettings_vivado_diff_fasm)
 
-function(xc7_iosettings_design MODE BOARD IOSTANDARD DRIVES SLEWS TERMS)
+function(xc7_iosettings_design)
```

Comment:
This function needs a documentation string.

Reply:
Added docstring.